src/main/java/ome/io/bioformats/BfPyramidPixelBuffer.java

```diff
@@ -440,7 +440,7 @@ protected void closeWriter() throws IOException
             try {
                 if (writerFile != null) {
                     try {
-                        FileUtils.moveFile(writerFile, readerFile);
+                        Files.move(writerFile.toPath(), readerFile.toPath());
```

Comment:
Just a caveat, the docs for nio.files state `The move fails if the target file exists, unless the REPLACE_EXISTING option is specified.` Might need to chat that is expected behaviour .

Reply:
Yep, I checked that before I made the change. That specific behaviour is identical to that of `FileUtils.moveFile()`:

 * https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#move-java.nio.file.Path-java.nio.file.Path-java.nio.file.CopyOption...-
 * https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FileUtils.html#moveFile(java.io.File,%20java.io.File)

There are likely other behavioral differences but I don't think they're particularly relevant.